cms/static/sass/views/_group-configuration.scss

```diff
@@ -19,206 +19,200 @@
+        .collection-description {
+          overflow: hidden;
+          text-overflow: ellipsis;
+        }
+      }
 
-            &:hover {
-              color: $blue;
+      .collection-items {
+        margin-left: $baseline;
+        margin-bottom: ($baseline*0.75);
 
-              .ui-toggle-expansion {
-                color: $blue;
-              }
-            }
+        .item {
+          @extend %t-copy-lead1;
+          padding: ($baseline/7) 0 ($baseline/4);
+          border-top: 1px solid $gray-l4;
+          white-space: nowrap;
+
+          &:first-child {
+            border-top: none;
           }
 
-          .groups {
-            margin-left: $baseline;
-            margin-bottom: ($baseline*0.75);
+          .name {
+            overflow: hidden;
+            text-overflow: ellipsis;
+            display: inline-block;
+            vertical-align: middle;
+            width: 75%;
+            margin-right: 5%;
+          }
+        }
+      }
 
-            .group {
-              @extend %t-copy-lead1;
-              padding: ($baseline/7) 0 ($baseline/4);
-              border-top: 1px solid $gray-l4;
-              white-space: nowrap;
+      .collection-details {
 
-              &:first-child {
-                border-top: none;
-              }
+        .actions {
+          @include transition(opacity .15s .25s ease-in-out);
+          opacity: 0.0;
+          position: absolute;
+          top: $baseline;
+          right: $baseline;
 
-              .group-name {
-                overflow: hidden;
-                text-overflow: ellipsis;
-                display: inline-block;
-                vertical-align: middle;
-                width: 75%;
-                margin-right: 5%;
-              }
+          .action {
+            display: inline-block;
+            vertical-align: middle;
+            margin-right: ($baseline/4);
 
-              .group-allocation {
-                display: inline-block;
-                vertical-align: middle;
-                width: 20%;
-                color: $gray-l1;
-                text-align: right;
-              }
+            .edit {
+              @extend %ui-btn-non-blue;
             }
-          }
 
-          .actions {
-            @include transition(opacity .15s .25s ease-in-out);
-            opacity: 0.0;
-            position: absolute;
-            top: $baseline;
-            right: $baseline;
+            .delete {
+              @extend %ui-btn-non;
 
-            .action {
-              display: inline-block;
-              vertical-align: middle;
-              margin-right: ($baseline/4);
-
-              .edit {
-                @include blue-button;
-                @extend %t-action4;
-              }
-
-              .delete {
-                @extend %ui-btn-non;
-
-                &.is-disabled {
-                  background-color: $gray-l3;
-                  color: $gray-l6;
-                }
+              &.is-disabled {
```

Comment:
doing some custom styling here to have the button look more button-y, but disabled. (otherwise it just looks like an icon hanging out)